design/dedicated-osd-pod.md

```diff
@@ -23,22 +23,34 @@ A more comprehensive discussions can be found at [this issue](https://github.com
 
 ## Proposal   
 
-We propose the following OSD startup change.
+We propose the following change to address the limitations.
+
+### Terms
+
+- OSD Discovery. A process that is given a device or directory path filter and use the filter to discover devices on the host. The process updates the OSD orchestration Configmap with the device paths and mark the orchestration status as "DeviceDiscovered". The process is created by Rook Operator and runs in a Kubernetes deployment on the storage node. This process is storage backend agnostic. Currently `rook osd` command implements this functionality.
```

Comment:
A few thoughts:
- This should be called "Device discovery" since it's backend agnostic.
- Instead of process, let's call it a pod. Otherwise it sounds like we're not in a K8s environment
- In the design doc i don't think it's important to mention a detail like `rook osd` for how it works today.

Reply:
i like how this device discovery is backend agnostic and will provide information about all the devices for nodes in the cluster, that any storage backend controller could consume